pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -65,7 +65,9 @@ type GCECloud struct {
 	service           *compute.Service
 	containerService  *container.Service
 	projectID         string
-	zone              string
+	region            string
+	instanceZone      string
+	managedZones      []string
```

Comment:
The meanings of these two zone constructs are not clear.  Please comment to clarify.